packages/mattermost-redux/src/reducers/entities/threads/counts.ts

```diff
@@ -22,6 +22,32 @@ function handleAllTeamThreadsRead(state: ThreadsState['counts'], action: Generic
     };
 }
 
+function isEqual(state: ThreadsState['counts'], action: GenericAction, unreads: boolean) {
```

Comment:
nit: function name could be a bit clearer, maybe:
```suggestion
function areUnreadsEqual(state: ThreadsState['counts'], action: GenericAction, unreads: boolean) {
```

Reply:
@jwilander  It's not just for unreads, but the whole counts reducer, so it could be used anywhere in this file,
so in my mind it's `isStateEqual`

Reply:
Ahh, got it